docs/runtimes/databricks.ipynb

```diff
@@ -93,6 +109,116 @@
     "    == \"kwargs: {'param1': 'value1', 'param2': 'value2'}\\n\"\n",
```

Comment:
this assert:
assert (\n",
    "    run.status.results[\"databricks_runtime_task\"][\"logs\"]\n",
    "    == \"kwargs: {'param1': 'value1', 'param2': 'value2'}\\n\"\n"

is no longer supported. please delete it.

Reply:
deleted